cudamapper/src/main.cu

```diff
@@ -775,3 +775,13 @@ int main(int argc, char* argv[])
 
     return 0;
 }
+
+} // namespace cudamapper
+} // namespace claragenomics
+
+/// \brief main function
+/// main function cannot be in a namespace so using this function to call actual main function
+int main(int argc, char* argv[])
```

Comment:
Why not just leave main in the global namespace and do `using namespace claragenomcs::cudamapper;` inside?

Reply:
discussed offline.